mobile/test/common/integration/base_client_integration_test.cc

```diff
@@ -199,6 +203,38 @@ void BaseClientIntegrationTest::createEnvoy() {
   engine_running.WaitForNotification();
 }
 
+uint64_t BaseClientIntegrationTest::getCounterValue(const std::string& name) {
+  uint64_t counter_value = 0UL;
+  uint64_t* counter_value_ptr = &counter_value;
+  absl::Notification counter_value_set;
+  EXPECT_EQ(ENVOY_SUCCESS,
+            EngineHandle::runOnEngineDispatcher(
+                rawEngine(), [counter_value_ptr, &name, &counter_value_set](Envoy::Engine& engine) {
+                  Stats::CounterSharedPtr counter =
+                      TestUtility::findCounter(engine.getStatsStore(), name);
+                  if (counter != nullptr) {
+                    *counter_value_ptr = counter->value();
+                  }
+                  counter_value_set.Notify();
+                }));
+  EXPECT_TRUE(counter_value_set.WaitForNotificationWithTimeout(absl::Seconds(5)));
+  return counter_value;
+}
+
+testing::AssertionResult BaseClientIntegrationTest::waitForCounterGe(const std::string& name,
```

Comment:
I agree, explicit would be better, but `Ge` is used throughout Envoy, via https://github.com/envoyproxy/envoy/blob/357bf0016ffa0102ed1fc3c51687eb23b9d2305d/test/test_common/utility.h#L261, so I mainly just wanted to be consistent so that the counter checks between Envoy and Envoy-Mobile read the same.